### PR TITLE
Adding VIND role for BCSC Account Creation

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

Adding VIND Role for BCSC Account creation

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Unit tests

## Does the change impact or break the Docker build?

- [ ] Yes
- [X] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/libs/efiling-cso-client/src/main/java/ca/bc/gov/open/jag/efilingcsoclient/CsoAccountServiceImpl.java [hunk 1]
```diff
@@ -186,6 +186,13 @@ private Client setCreateAccountClientDetails(CreateAccountRequest createAccountR
         roleFile.setRegisteredClientRoleCd("FILE");
         roles.add(roleFile);
 
+        if (identityProvider.equalsIgnoreCase("BCSC")) {
```
Comment (KDNeufeld) [inline src/backend/libs/efiling-cso-client/src/main/java/ca/bc/gov/open/jag/efilingcsoclient/CsoAccountServiceImpl.java:189]: Magic string. Possibly extract to constant.

File: src/backend/libs/efiling-cso-client/src/main/java/ca/bc/gov/open/jag/efilingcsoclient/CsoAccountServiceImpl.java [hunk 2]
```diff
@@ -186,6 +186,13 @@ private Client setCreateAccountClientDetails(CreateAccountRequest createAccountR
...
+            RoleAssignment roleVind = new RoleAssignment();
+            roleVind.setActiveYn(true);
+            roleVind.setRegisteredClientRoleCd("VIND");
```
Comment (KDNeufeld) [inline src/backend/libs/efiling-cso-client/src/main/java/ca/bc/gov/open/jag/efilingcsoclient/CsoAccountServiceImpl.java:192]: Magic string. Possibly extract to constant.

**Review by KDNeufeld — CHANGES_REQUESTED** [review]
Excellent work.  Just a small change regarding constants.

File: src/backend/libs/efiling-cso-client/src/main/java/ca/bc/gov/open/jag/efilingcsoclient/CsoAccountServiceImpl.java (hunk 1 above)
Comment (TayGov) [reply src/backend/libs/efiling-cso-client/src/main/java/ca/bc/gov/open/jag/efilingcsoclient/CsoAccountServiceImpl.java]: Yes, I will refactor the method and all the constants that should be there.

**Review by alexjoybc — APPROVED** [review]
Everything is Awesome

**Review by KDNeufeld — APPROVED** [review]
Great work!

**Pull request merged**